source/memory.tex

```diff
@@ -5392,7 +5392,8 @@
 
 \pnum
 A specialization of class template \tcode{pmr::polymorphic_allocator} meets
-the \oldconcept{Allocator} requirements\iref{allocator.requirements.general}.
+the \oldconcept{Allocator} requirements\iref{allocator.requirements.general}
+if its template argument is a \cv-unqualified object type.
```

Comment:
The second paragraph needs a similar change: "All specializations meet the allocator completeness requirements" implies that all specializations meet the _`Cpp17Allocator`_ requirements.